Makefile

```diff
@@ -1,8 +1,17 @@
-
-test:
-	rake spec
-
-build:
-	gem build ./analytics-ruby.gemspec
-
-.PHONY: test build
+
+# Install any tools required to build this library, e.g. Node/Yarn etc.

```

Comment:
> e.g. Node/Yarn etc.

Could replace this with a Ruby specific example I think (e.g. Ruby, Bundler etc.)

Reply:
Fixed.